types/d3-array/index.d.ts

```diff
@@ -1075,3 +1075,19 @@ export function thresholdFreedmanDiaconis(values: ArrayLike<number | undefined>,
 export function thresholdScott(values: ArrayLike<number | undefined>, min: number, max: number): number; // of type ThresholdCountGenerator
 
 export function thresholdSturges(values: ArrayLike<number | undefined>): number; // of type ThresholdCountGenerator
+
```

Comment:
Should these not be extracted as separate types in a new internmap project (as it is a separate repo: https://github.com/mbostock/internmap)? Not really sure how likely it is that someone will use these outside of here, so unsure if worth it

Reply:
I thought about it, but it didn't seem worth the overhead. If other types need to start depending on it then we should definitely pull it out, but the types are so simple that I'd rather not go to the trouble if that sounds reasonable to you.

Reply:
That sounds very pragmatic indeed, so I guess we can fly like this for this iteration. Should the need arise to split it, it can be done at a later stage